deploy/examples/external/dashboard-external-http.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: rook-ceph-mgr-dashboard-external-http
+  namespace: rook-ceph # namespace:cluster
```

Comment:
why not to add the symlink

Reply:
Do I also create symlink for `dashboard-external-https.yaml `file?